This is suitable for upstream but the development is dead.

--- a/libdaemon/dfork.c	2024-03-21 09:43:47.929171407 +0100
+++ b/libdaemon/dfork.c	2024-03-21 09:43:41.946866354 +0100
@@ -415,7 +415,7 @@
                 daemon_log(LOG_ERR, "select() failed while waiting for return value: %s", strerror(errno));
             else {
                 errno = ETIMEDOUT;
-                daemon_log(LOG_ERR, "Timeout reached while wating for return value");
+                daemon_log(LOG_ERR, "Timeout reached while waiting for return value");
             }
 
             return -1;
